spack/local/packages/palace/package.py

```diff
@@ -22,24 +32,59 @@ class Palace(CMakePackage, CudaPackage, ROCmPackage):
     # Note: 'rocm' and 'amdgpu_target' variants are added by the ROCmPackage
     variant("shared", default=True, description="Build shared libraries")
     variant("int64", default=False, description="Use 64 bit integers")
-    variant("openmp", default=False, description="Use OpenMP for shared-memory parallelism")
     variant(
-        "superlu-dist", default=True, description="Build with SuperLU_DIST sparse direct solver"
+        "openmp", default=False, description="Use OpenMP for shared-memory parallelism"
+    )
+    variant(
+        "superlu-dist",
+        default=True,
+        description="Build with SuperLU_DIST sparse direct solver",
+    )
+    variant(
+        "strumpack",
+        default=False,
+        description="Build with STRUMPACK sparse direct solver",
+    )
+    variant(
+        "sundials",
+        default=True,
+        description="Build with SUNDIALS differential/algebraic equations solver",
     )
-    variant("strumpack", default=False, description="Build with STRUMPACK sparse direct solver")
-    variant("sundials", default=True, description="Build with SUNDIALS differential/algebraic equations solver")
     variant("mumps", default=False, description="Build with MUMPS sparse direct solver")
     variant("slepc", default=True, description="Build with SLEPc eigenvalue solver")
     variant("arpack", default=False, description="Build with ARPACK eigenvalue solver")
-    variant("libxsmm", default=True, description="Build with LIBXSMM backend for libCEED")
     variant("magma", default=True, description="Build with MAGMA backend for libCEED")
     variant(
         "gslib",
         default=True,
         description="Build with GSLIB library for high-order field interpolation",
     )
 
-    # Dependencies
+    # TODO: Apply patches for all packages...
+    # TODO: We should actually use these as externals...
```

Comment:
Seems like many of our patches aren't necessary to build our deps. Perhaps testing will be necessary as we move to using spack built dependencies.